src/ngx_http_lua_balancer.c

```diff
@@ -286,21 +440,24 @@ ngx_http_lua_balancer_get_peer(ngx_peer_connection_t *pc, void *data)
 
     ctx->context = NGX_HTTP_LUA_CONTEXT_BALANCER;
 
+    bp->kpool = NULL;
```

Comment:
"kpool" may not be a good name. How about use "mpool" for "memory pool" and "cpool" for "connection pool" consistently? The latter names can be used NOT just for this module.